scripts/data_utils.py

```diff
@@ -277,6 +317,30 @@ def get_frequency(items: Iterable, match: Union[Any, Container[Any]]) -> float:
     return n_matching / len(s)
 
 
+def aggregate_vit_predictions(df: pd.DataFrame, model_name: str, flops: float = None) -> pd.DataFrame:
+    """Aggregate ViT predictions into performance metrics.
+    
+    Args:
+        df (pd.DataFrame): DataFrame containing ViT predictions with 'real_class', 
+            'predicted_class', 'real_quaternion', and 'predicted_quaternion' columns.
+        model_name (str): Name of the ViT model (e.g., 'ViT-B/16').
+        flops (float, optional): FLOPs for this model. If not provided, will be NaN.
+        
+    Returns:
+        pd.DataFrame: DataFrame with one row containing model name, accuracy, 
+            rotation error, and FLOPs.
+    """
+    accuracy = compute_vit_accuracy(df) * 100  # Convert to percentage
+    rotation_error = compute_vit_rotation_error(df)
```

Comment:
Hey Hojae, for Figure 7 we only use the correct predictions to calculate the rotation error for ViT (and Monty). It doesn't seem like we are doing that here for Figure 8?